server/src/dbt_execution/DbtRpcCompileJob.ts

```diff
@@ -12,6 +12,8 @@ export class DbtRpcCompileJob extends DbtCompileJob {
   static readonly NETWORK_ERROR = 'Network error';
   static readonly JOB_IS_NOT_COMPLETED = 'Job is still not completed';
 
+  static readonly NO_RESULT_FROM_COMPILER = ' ';
```

Comment:
RegExp ?

Reply:
What do you mean?

Reply:
If we can consider any sequence of spaces as "no results" ?

Reply:
Aaa, no, this one space result we return and this means that we have no result from compiler. Not sure if there is any better name)